client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningClinical.tsx

```diff
@@ -233,18 +233,21 @@ const ContactQuestioningClinical: React.FC<Props> = (props: Props): JSX.Element
                                         disabled={isFieldDisabled || (shouldDisableIdByReopen && interactedContact.doesNeedIsolation === true)}
                                         test-id='doesNeedIsolation'
                                         onChange={(event, booleanValue) => {
-                                            if (booleanValue !== null) {
+                                            if (booleanValue == false) {
```

Comment:
Why not !Boolean(BooleanValue)

Reply:
Because Boolean value is a weird name, and i think it’s less readable like that